### PR TITLE
add-new-definitions

### PR DESCRIPTION
**Pull request by eltrubetskaya**

### CODE REVIEW

**Comment by spolischook** [comment]
Nice work. Please request review when you finish and fix all tests

File: src/AppBundle/Entity/PriceCategory.php [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+     */
+    protected $venue;
+
+    /**
+     * @var Collection|Seat[]
+     *
+     * @ORM\OneToMany(
+     *     targetEntity="AppBundle\Entity\Seat",
+     *     mappedBy="priceCategory",
+     *     cascade={"persist", "remove"}
```
Comment (spolischook) [inline src/AppBundle/Entity/PriceCategory.php]: I don't think that we need remove seats during remove price category

Comment (eltrubetskaya) [inline src/AppBundle/Entity/PriceCategory.php]: ) of course not. Thank you

**Comment by sobchenko** [comment]
@eltrubetskaya Great work!!! Please, do not forget to create migration!

**Pull request merged**